app/models/ability.rb

```diff
@@ -240,20 +232,20 @@ def signed_in_with_volunteers_coordinator_role(user)
     # ids of all the conferences for which the user has the 'volunteers_coordinator' role
     conf_ids_for_volunteers_coordinator = []
     conf_ids_for_volunteers_coordinator =
-        Conference.with_role(:volunteer_coordinator, user).pluck(:id) if user.has_role? :volunteer_coordinator, :any
+        Conference.with_role(:volunteers_coordinator, user).pluck(:id) if user.has_role? :volunteers_coordinator, :any
 
     can :manage, Vposition, conference_id: conf_ids_for_volunteers_coordinator
     can :manage, Vday, conference_id: conf_ids_for_volunteers_coordinator
+
     # Abilities for Role (Conference resource)
     can :index, Role
-
-    can :manage, Role do |role|
+    can :show, Role do |role|
       role.resource_type == 'Conference' && (conf_ids_for_volunteers_coordinator.include? role.resource_id)
     end
 
-    # Can add or remove users from role, when user has that same role for the conference
+    # Can add or remove users from role and edit role description, when user has that same role for the conference
     # Eg. If you are member of the CfP team, you can add more CfP team members (add users to the role 'CfP')
-    can :toggle_user, Role do |role|
+    can [:toggle_user, :update, :edit], Role do |role|
       role.resource_type == 'Conference' &&
```

Comment:
Because we could use roles on another resource (even though we are not currently). This will keep our code correct no matter what we do in the future.